mne/time_frequency/multitaper.py

```diff
@@ -230,7 +230,7 @@ def _csd_from_mt(x_mt, y_mt, weights_x, weights_y):
     return csd
 
 
-def _mt_spectra(x, dpss, sfreq, n_fft=None):
+def _mt_spectra(x, dpss, sfreq, remove_dc=True, n_fft=None):
```

Comment:
```suggestion
def _mt_spectra(x, dpss, sfreq, n_fft=None, remove_dc=True):
```

this is probably the cause of the test failures: because those other functions are passing args, not kwargs, and you inserted your new kwarg before `n_fft` in `_mt_spectra`. Put your new kwarg at the end.

Reply:
ok! sounds good

Reply:
with this change there are no more errors, thanks